### PR TITLE
update document to use tiflash in tispark

### PR DESCRIPTION
**Pull request by Joyinqin**
### What is changed, added or deleted? (Required)

update document to use tiflash in tispark

parameter `spark.tispark.use.tiflash` is replaced with `spark.tispark.isolation_read_engines`.
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5030
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Joyinqin** [comment]
needs-cherry-pick-3.1, needs-cherry-pick-4.0, size/small,translation/from-docs-cn

File: tiflash/use-tiflash.md [hunk 1]
```diff
@@ -193,7 +193,7 @@ In the above three ways of reading TiFlash replicas, engine isolation specifies
 
 ## Use TiSpark to read TiFlash replicas
 
-Currently, you can use TiSpark to read TiFlash replicas in a method similar to the engine isolation in TiDB. This method is to configure the `spark.tispark.use.tiflash` parameter to `true` (or `false`).
+Currently, you can use TiSpark to read TiFlash replicas in a method similar to the engine isolation in TiDB. This method is to configure the `spark.tispark.isolation_read_engines` parameter. The parameter value defaults to `tikv,tiflash`, which means automatically reading data from TiFlash or TiKV according to CBO. If you set the parameter value to `tiflash`, it means that data is forcibly read from TiFlash.
```
Comment (TomShawn) [inline tiflash/use-tiflash.md:196]:
```suggestion
Currently, you can use TiSpark to read TiFlash replicas in a method similar to the engine isolation in TiDB. This method is to configure the `spark.tispark.isolation_read_engines` parameter. The parameter value defaults to `tikv,tiflash`, which means that TiDB reads data from TiFlash or from TiKV according to CBO's selection. If you set the parameter value to `tiflash`, it means that TiDB forcibly reads data from TiFlash.
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**